kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/BaseCommand.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2019. Tesla Motors, Inc. All rights reserved.
+ */
+
+package com.tesla.data.topic.enforcer;
+
+import com.beust.jcommander.IStringConverter;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+import com.beust.jcommander.Parameters;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * All topic enforcement related CLI tools are an extension of {@link BaseCommand}.
+ */
+@Parameters(commandDescription = "Validate config")
+class BaseCommand {
```

Comment:
should this ever be run? if not, probably should make abstract and remove the `@parameters`

Reply:
Yeah, found useful in few scenarios just to validate the config.